packages/components/src/Modal.tsx

```diff
@@ -29,15 +29,20 @@ const ModalOuter = styled<ToggleDisplayEl & TextAlignProps, "div">("div")`
 interface ModalInnerProps {
   width?: number;
   padding?: string;
+  fullScreen?: boolean;
```

Comment:
Just curious, what's the difference between this and `@joincivil/components/src/FullScreenModal`?

Reply:
didn't know that existed